### PR TITLE
Add grunt

### PR DESCRIPTION
**Pull request by wytrych**

### CODE REVIEW

**Comment by wytrych** [comment]
@incuna/js @incuna/frontend Please review

File: web/eslint.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "sourceType": "module",
+        "ecmaFeatures": {
+            "modules": true
+        }
+    },
+    "extends": "eslint:recommended",
```
Comment (Minglee01) [inline web/eslint.json]: i think these rules are wrong.
http://eslint.org/docs/rules/curly

Comment (Minglee01) [inline web/eslint.json]: also you still need a jscrc file for now. ass all the rules arent intergrated yet @grahamgilchrist ?

Comment (Minglee01) [inline web/eslint.json]:
```
Configuring Rules
ESLint comes with a large number of rules. You can modify which rules your project uses either using configuration comments or configuration files. To change a rule setting, you must set the rule ID equal to one of these values:

"off" or 0 - turn the rule off
"warn" or 1 - turn the rule on as a warning (doesn’t affect exit code)
"error" or 2 - turn the rule on as an error (exit code is 1 when triggered)
```

from http://eslint.org/docs/user-guide/configuring

Comment (wytrych) [inline web/eslint.json]: It must have change lately. Do you have a project where these rules are up to date?

Comment (Minglee01) [inline web/eslint.json]: its been like it for at least a month
https://github.com/incuna/angular-feincms/blob/master/.eslintrc

Comment (wytrych) [inline web/eslint.json]: A month isn't that long :) When I was doing it for bi it was different. I'll change

Comment (Minglee01) [inline web/eslint.json]: well the example i listed was from june

Comment (Minglee01) [inline web/eslint.json]: but if you can test the rules failing as they are then its ok

Comment (Minglee01) [inline web/eslint.json]: The version im using is 18 if  bi is using this or a later version the rules may need testing.

**Comment by wytrych** [comment]
@Minglee01 updated

File: web/.eslintrc.es [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "extends": [
+        ".eslintrc.main"
```
Comment (henrahmagix) [inline web/.eslintrc.es]: niiiiiiiiiiiiiiiiiiiice

File: web/Gruntfile.js [hunk 3]
```diff
@@ -22,7 +23,10 @@ module.exports = function (grunt) {
             srcScriptsDir: '<%= config.scriptsDir %>/src',
             compiledScriptsDir: '<%= config.scriptsDir %>/compiled-es5',
 
-            lintFiles: ['<%= config.srcScriptsDir %>', 'Gruntfile.js']
+            lintFiles: {
+                node: ['Gruntfile.js'],
+                es: ['<%= config.srcScriptsDir %>']
```
Comment (henrahmagix) [inline web/Gruntfile.js]: es -> browser?

Comment (wytrych) [inline web/Gruntfile.js]: it's probably worth keeping consistent, it's `es` everywhere

**Comment by wytrych** [comment]
@henrahmagix updated

**Comment by henrahmagix** [comment]
👍

**Pull request merged**